cnf-tests/testsuites/e2esuite/security/tuning.go

```diff
@@ -137,4 +137,56 @@ var _ = Describe("[tuningcni]", func() {
 				Expect(err).ToNot(HaveOccurred())
 			})
 	})
+
+	Context("sysctl allowlist update", func() {
+		var originalSysctls = ""
+
+		BeforeEach(func() {
+			cm, err := client.Client.ConfigMaps("openshift-multus").Get(context.TODO(), "cni-sysctl-allowlist", metav1.GetOptions{})
+			Expect(err).ToNot(HaveOccurred())
+			var ok bool
+			originalSysctls, ok = cm.Data["allowlist.conf"]
+			Expect(ok).To(BeTrue())
+		})
+
+		AfterEach(func() {
+			updateAllowlistConfig(originalSysctls)
+		})
+
+		It("should start a pod with custom sysctl only after adding sysctl to allowlist", func() {
+			macvlanNadName := "macvlan-nad1"
+			updatedSysctls := originalSysctls + "\n" + "^net.ipv4.conf.IFNAME.accept_local$"
+
+			podSysctls, err := networks.SysctlConfig(map[string]string{"net.ipv4.conf.IFNAME.accept_local": "1"})
+			macVlandNad, err := networks.NewNetworkAttachmentDefinitionBuilder(TestNamespace, macvlanNadName).WithMacVlan().WithTuning(podSysctls).Build()
+			Expect(err).ToNot(HaveOccurred())
+			err = client.Client.Create(context.Background(), macVlandNad)
+			Expect(err).ToNot(HaveOccurred())
+
+			podDefinition := pods.DefineWithNetworks(TestNamespace, []string{fmt.Sprintf("%s/%s", TestNamespace, macvlanNadName)})
+			pod, err := client.Client.Pods(TestNamespace).Create(context.Background(), podDefinition, metav1.CreateOptions{})
+			Expect(err).ToNot(HaveOccurred())
+			Consistently(func() corev1.PodPhase {
+				checkPod, err := client.Client.Pods(TestNamespace).Get(context.Background(), pod.Name, metav1.GetOptions{})
+				Expect(err).ToNot(HaveOccurred())
+				return checkPod.Status.Phase
```

Comment:
please check the even here that they pending is because there is a multus error with the tuning pluign